rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -190,7 +190,14 @@ target_include_directories(${_target_name_lib}
   ${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_py
   ${PythonExtra_INCLUDE_DIRS}
 )
-if(APPLE OR WIN32)
+
+# Check if numpy is in the include path
+find_file(_numpy_h numpy/numpyconfig.h
+  PATHS ${PythonExtra_INCLUDE_DIRS}
+  ${INCLUDE_INSTALL_DIR}
```

Comment:
Why also checking `INCLUDE_INSTALL_DIR`?

Reply:
I figured in case numpy was installed to a non-standard path, but it's probably safe to exclude it here.